README.md

```diff
@@ -9,15 +9,28 @@ VAST students can use Vaasu bot to get the attendance details by logging in usin
 
 The credentials are encrypted and stored in the database.
 
-## Installation
 
-```
-pip3 install python-dotenv python-telegram-bot requests
-```
+## Setup
+
+#### Clone the repo
```

Comment:
Instead of headings, it's better to use bullet points with `*` [I assume you've seen markdown cheatsheet]